service/src/main/resources/application.yaml

```diff
@@ -29,25 +47,25 @@ quarkus:
       generation: update
   qpid-jms:
     url: "${UMB_BROKER_URL}"
-  log:
-    level: INFO
-    category:
-      "org.jboss.sbomer":
-        level: DEBUG
-      "com.github.tennaito.rsql":
-        level: WARN
-    console:
-      format: "%d{HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e mdc:[%X]%n"
+
   oidc:
+    auth-server-url: "${AUTH_SERVER_URL}"
```

Comment:
Ok this one was actually removed on purpose to make the devservices for Keycloak usable in dev mode, with this commit https://github.com/project-ncl/sbomer/commit/1a5a5e5e91d42f30714e26c9fb195baaf1d10753.
We can leave it but we will need to remove it once we will be back testing Keycloak and devservices

Reply:
Removed!